ansible_collections/arista/avd/roles/eos_designs/doc/platform-settings-v4.0.md

```diff
@@ -29,8 +29,10 @@ platform_settings:
 
 # Set Hardware Speed Groups per Platform
 platform_speed_groups:
-  < platform >:
-    < speed >: [ < speed_group >, < speed_group > ]
+  - name: < platform >
```

Comment:
since the key name is not just `platforms` I think we should use the primary key `platform` to be clear. Thanks.